src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+  setCurrentTab,
+}) => {
+  const onTabHandler = (tab) => {
+    if (tab.id !== selectedTabId) {
+      onTabSelected(tab);
+      setCurrentTab(tab);
+    }
+  };
+
+  return (
+    <ul>
+      {tabs.map(tab => (
+        <li
+          className={cn({ 'is-active': tab.id === selectedTabId })}
+          data-cy="Tab"
+          key={tab.id}
+        >
+          <a
+            href={`#${tab.id}`}
+            data-cy="TabLink"
+            onClick={() => onTabHandler(tab)}
+          >
+            {tab.title}
+          </a>
+        </li>
+      ))}
+    </ul>
```

Comment:
It seems like there are in `Tabs` component should be not only `<ul></ul>` with tabs, but all this `div` with `data-cy="TabsComponent"`. 
Maybe it helps you to pass all tests.